src/test/java/PlaylistTests.java

```diff
@@ -9,8 +9,37 @@ public class PlaylistTests extends BaseTest {
     public void createPlaylist() throws InterruptedException {
         HomePage homePage = new LoginPage(driver).login();
         int initialNumberOfPlaylists = homePage.getNumberOfPlaylist();
-        homePage.createPlaylist("name");
+        homePage.createPlaylist("Super Songs");
         int updatedNumberOfPlaylists = homePage.getNumberOfPlaylist();
-        Assert.assertEquals(updatedNumberOfPlaylists, initialNumberOfPlaylists+1, "Playlist not created");
+        Assert.assertEquals(updatedNumberOfPlaylists, initialNumberOfPlaylists + 1, "Playlist not created");
     }
-}
\ No newline at end of file
+
+    @Test
+
+    public void createPlaylistWithSameName() throws InterruptedException {
+        HomePage homePage = new LoginPage(driver).login();
+        String msg = homePage.getDuplicatePlaylist("Super Songs");
```

Comment:
This variable should be renamed to something more descriptive , like` actualResultDuplicatePlaylist`
 `String actualResultDuplicatePlaylist = homePage.getDuplicatePlaylist("Super Songs");`